camp/camp-brooklyn/src/main/java/org/apache/brooklyn/camp/brooklyn/spi/dsl/parse/DslParser.java

```diff
@@ -52,7 +53,29 @@ public Object next() {
         if (index >= expression.length())
             throw new IllegalStateException("Unexpected end of expression to parse, looking for content since position "+start);
         
-        if (expression.charAt(index)=='"') {
+        String multilineTemplatePrefix = "$brooklyn:template";
```

Comment:
Seems odd to have just `$brooklyn:template` and then the line break - can this not be done something like
```
$brooklyn:template: |
   my ${template} here
```